agent/internal/options/options.go

```diff
@@ -221,14 +220,6 @@ const (
 	PodmanContainerRuntime      = "podman"
 )
 
-// SingularityOptions configures how we interact with Singularity.
-type SingularityOptions struct {
-	// AllowNetworkCreation allows the agent to use `singularity run`'s `--net` option, which sets
-	// up and launches containers into a new network namespace. Disabled by default since this
-	// requires root or a suid installation with /etc/subuid --fakeroot.
-	AllowNetworkCreation bool `json:"allow_network_creation"`
-}
-
 // PodmanOptions configures how we interact with podman.
 type PodmanOptions struct {
```

Comment:
I am currently leaving this out. Would remove it if not needed in the next [subtask](https://hpe-aiatscale.atlassian.net/browse/RM-352)